content/graphing/infrastructure/livecontainers.md

```diff
@@ -95,6 +95,16 @@ It would be easy to pivot by ECS `ecs_task_name` and `ecs_task_version` and unde
 
 {{< img src="graphing/infrastructure/livecontainers/tasksxversion.png" alt="Tasks x version" responsive="true" style="width:80%;">}}
 
+## Scatterplot for Live Containers
+
+The scatterplot is designed for high volume data analysis and, with intelligent grouping, can help you explore all of the containers across your entire environment. With this new visualization, you are able to compare these metrics with one another to better understand the performance of your containers.
```

Comment:
r / `	The scatterplot is designed for high volume data analysis and, with intelligent grouping, can help you explore all of the containers across your entire environment. With this new visualization, you are able to compare these metrics with one another to better understand the performance of your containers.` / `The scatterplot is designed to compare your containers metrics with one another to better understand their performances`